fedora/client/openidbaseclient.py

```diff
@@ -154,6 +160,20 @@ def __init__(self, base_url, login_url=None, useragent=None, debug=False,
 
         # python-requests session.  Holds onto cookies
         self._session = requests.session()
+
+        # Also hold on to retry logic.
+        # http://www.coglib.com/~icordasc/blog/2014/12/retries-in-requests.html
+        if retries is not None:
+            prefixes = ['http://', 'https://']
+            for prefix in prefixes:
+                self._session.mount(prefix, requests.adapters.HTTPAdapter(
+                    max_retries=Retry(
+                        total=retries,
+                        status_forcelist=[500],
+                        backoff_factor=retry_backoff_factor,
+                    ),
+                ))
+
```

Comment:
Correct! Which is different than how #152 does it.  the requests API kind of imposes that on us, given how you mount an Adapter on the session, but I like it in terms of the python-fedora API too.  it's more simple.